matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/crypto/store/db/Helper.kt

```diff
@@ -51,8 +50,7 @@ fun <T : RealmObject> doRealmQueryAndCopy(realmConfiguration: RealmConfiguration
  */
 fun <T : RealmObject> doRealmQueryAndCopyList(realmConfiguration: RealmConfiguration, action: (Realm) -> Iterable<T>): Iterable<T> {
     return Realm.getInstance(realmConfiguration).use { realm ->
-        val result = action.invoke(realm)
-        realm.copyFromRealm(result)
+        action.invoke(realm).let { realm.copyFromRealm(it) }
```

Comment:
I find it less elegant...

Reply:
Yes, I wanted to keep the reading from left to right, because I could have inline the parameter of copyFromRealm...